### ISSUE TITLE
Tree drawing error: rendering overflow

### ISSUE THREAD

**Issue by a1an77** [veidenberg/wasabi#15]
With the attached dataset, the tree drawing produces a weird result (see orange circle in attached image). It seems hel4 sequence does not even belong to tree. I suspect somehow a portion of the tree gets drawn off the right border and is hence not shown.

![tree_bug](https://user-images.githubusercontent.com/4610478/45157106-6e880a00-b1e0-11e8-8559-3cfcdc31e623.png)

**Comment by a1an77** [comment]
Here the missing attachments (added txt suffix to let them be accepted by github)
[tree_bug_all.fasta.txt](https://github.com/veidenberg/wasabi/files/2357000/tree_bug_all.fasta.txt)
[tree_bug_all.tree.txt](https://github.com/veidenberg/wasabi/files/2357001/tree_bug_all.tree.txt)

**Comment by veidenberg** [comment]
I noticed that leafs with very small branch lengths might get drawn over tree canvas borders. I modified floating point number rounding in tree drawing, hope it helps. The issue also fixes itself when initial tree canvas width is wider than default (e.g. when only tree is imported).

![tree_screenshot](https://user-images.githubusercontent.com/1215700/47167026-36f39e00-d306-11e8-9fdb-e6740841243b.png)
